templates/cart.liquid

```diff
@@ -18,10 +18,10 @@
         <div class="hidden-xs hidden-sm col-md-12 hidden-print">
             <div class="page-header clearfix">
                 <ol class="nav nav-pills pull-left">
-                    <li role="presentation" ng-repeat="step in checkout.wizard.steps" ng-class="{'active': step.isActive}">
-                        <a href="" ng-click="checkout.wizard.goToStep(step)">
-                            <span class="h1 text-middle" ng-bind="$index + 1"></span>
-                            <u class="h2 text-middle" ng-bind="step.title"></u>
+                    <li role="presentation" ng-repeat="step in checkout.wizard.steps" ng-class="{'checkout-wizard-active-step': step.isActive, 'disabled': !step.isActive}">
+                        <a href="">
+                            <span class="h2 text-middle" ng-bind="$index + 1"></span>
+                            <u class="h2 text-decoration-none text-middle" ng-class="{'text-default-color': !step.isActive}" ng-bind="step.title"></u>
```

Comment:
Use `span` instead and you will not need text-decoration-none

Reply:
done